prettyblocks.php

```diff
@@ -43,6 +43,17 @@ class PrettyBlocks extends Module implements WidgetInterface
         ],
     ];
 
+    public $hooks = [
+        'displayHeader',
+        'displayHome',
+        'displayLeftColumn',
+        'displayRightColumn',
+        'displayFooter',
+        'displayBlockZone',
```

Comment:
`displayBlockZone` hook doesn’t exist.

Reply:
Yes, I have taken the hook that was grafted in before my modification, but it can be removed if it is not used.

Can you confirm @PrestaSafe @micka-fdz ?

Reply:
Hi and thansk for this PR, yes, i think it was for testing, this hook can be removed.